Missing dependency in `@material-ui/labs`

<!-- Provide a general summary of the issue in the Title above -->

<!--
  Thank you very much for contributing to Material-UI by creating an issue! ❤️
  To avoid duplicate issues we ask you to check off the following list.
-->

<!-- Checked checkbox should look like this: [x] -->

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯

<!-- Describe what happens instead of the expected behavior. -->
Typescript fails to build with the following error:
```
./node_modules/@material-ui/lab/SpeedDial/SpeedDial.d.ts:4:33 - error TS7016: Could not find a declaration file for module 'react-transition-group/Transition'. './node_modules/react-transition-group/cjs/Transition.js' implicitly has an 'any' type.

```

## Expected Behavior 🤔

<!-- Describe what should happen. -->
Typescript builds successfully.

## Steps to Reproduce 🕹

<!--
  Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
  Include code to reproduce, if relevant (which it most likely is).

  You should use the official codesandbox template as a starting point: https://material-ui.com/r/issue-template

  If you have an issue concerning TypeScript please start from this TypeScript playground: https://material-ui.com/r/ts-issue-template

  Issues without some form of live example have a longer response time.
-->

Steps:

1. In a project that has material-ui installed
2. Run the a typescript build

## Context 🔦

<!--
  What are you trying to accomplish? How has this issue affected you?
  Providing context helps us come up with a solution that is most useful in the real world.
-->

## Your Environment 🌎

<!--
  Include as many relevant details about the environment with which you experienced the bug.
  If you encounter issues with typescript please include version and tsconfig.
-->

| Tech        | Version |
| ----------- | ------- |
| Material-UI | v4.9.9  |
| Material-UI/lab | v4.0.0-alpha.51  |
| React       | 16.13.0        |
| TypeScript  |  3.8.3       |

--- a/packages/material-ui-lab/src/SpeedDial/SpeedDial.d.ts
+++ b/packages/material-ui-lab/src/SpeedDial/SpeedDial.d.ts
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { StandardProps } from '@material-ui/core';
 import { FabProps } from '@material-ui/core/Fab';
-import { TransitionProps } from 'react-transition-group/Transition';
-import { TransitionHandlerProps } from '@material-ui/core/transitions';
+import { TransitionHandlerProps, TransitionProps } from '@material-ui/core/transitions';
 
 export type CloseReason = 'toggle' | 'blur' | 'mouseLeave' | 'escapeKeyDown';
 export type OpenReason = 'toggle' | 'focus' | 'mouseEnter';

